src/EventListener/InitializeSystemListener.php

```diff
@@ -151,6 +167,9 @@ private function boot($routeName, $basePath)
         // Log PHP errors
         $this->iniSet('error_log', $this->rootDir . '/system/logs/error.log');
 
+        // Support legacy Session access
+        $this->initializeLegacySessionAccess();
```

Comment:
Why is the method call here and not in line 163, when `$this->startSession()` used to be?